### PR TITLE
feat: use default value for quotes (double-quotes)

### PR DESCRIPTION
**Pull request by kevgo**
Before I apply Prettier formatting consistently to all Ory repos (which will change a ton of files), I want to propose to remove the single-quote override for Prettier. Reasons:

- Most languages we use here only work with double-quotes: Go, JSON, HTML, CSS, English.
- Most JS tooling we use prefers double-quotes: Prettier, ESLint.
- TS and YML can use either form but [chose to deviate from the other languages we use](https://youtu.be/vIJHp_y2PE8). That feels elitist 🙂 
- I see consistency in code style across Ory as more important than conforming to (arbitrary) external style standards. Since we are a Go company, let's make our JS style match that of Go: no semi and double quotes for strings.
- Most of the YML at Ory already uses double-quotes, let's not change that.
- With the current config, TSX often has [both types of quotes on a single line](https://github.com/ory/docs/blob/master/docs/hydra/guides/consent.mdx?plain=1#L84). That looks odd.

I am therefore proposing to remove the `singleQuote: true` override from the Ory-wide Prettier config. I think the benefit of removing pointless edge cases is worth the change. Curious what you think! No hard feelings if anybody sees it differently.

### CODE REVIEW

File: prettier.config.ts [hunk 1]
```diff
@@ -1,10 +1,9 @@
-import * as prettier from 'prettier'
+import * as prettier from "prettier"
 
 const options: prettier.Options = {
-  proseWrap: 'always',
+  proseWrap: "always",
   semi: false,
-  singleQuote: true,
-  trailingComma: 'none'
+  trailingComma: "none"
```
Comment (zepatrik) [inline prettier.config.ts]: I think this would also be more consistent with go formatting.

```suggestion
  trailingComma: "all",
```

**Review by zepatrik — COMMENTED** [review]
I want to remind everyone here that

> [By far the biggest reason for adopting Prettier is to stop all the ongoing debates over styles.](
https://prettier.io/docs/en/option-philosophy.html)

Regardless, we never yet had a debate about this but just used these options for years now. Your arguments make sense, and I actually don't care about the specifics, as long as it is consistent.

So from my side it would be fine to update these values once and follow gofmt as much as possible is a good target.

File: prettier.config.ts (hunk 1 above)
Comment (kevgo) [reply prettier.config.ts]: Excellent idea. This also minimizes diffs when adding or removing values from arrays.

**Review by zepatrik — APPROVED** [review]

**Comment by kevgo** [comment]
Thanks, 100% agree! The larger roadmap towards ending style debates through Prettier is:

1. remove configuration overrides that make code style less consistent across our code bases (this PR)
2. ensure consistent formatting in all repos via CI (coming PRs)
3. better support for people who are not familiar with Prettier

**Comment by zepatrik** [comment]
If you want to release this to npm, just create a release through github. But I guess you already figured that out :wink:

**Comment by kevgo** [comment]
That would have been my next question! I was assuming something like that from going through the release scripts and Git history but wanted to double-check with you. Thanks for reading my mind! 🙂

**Pull request merged**